packages/modula-examples/src/render_nav.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env browser */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function renderLi(exampleTitle){
```

Comment:
better to use more meaningful name from end user perspective instead of saying "Li", "Ul" which is pure implementation detail